internal/pkg/flink/internal/store/store_test.go

```diff
@@ -363,32 +363,66 @@ func (s *StoreTestSuite) TestParseSETStatement() {
 	key, value, _ = parseSetStatement("set 'key= \nvalue'	")
 	assert.Equal(s.T(), "", key)
 	assert.Equal(s.T(), "", value)
+
+	key, value, _ = parseSetStatement(`set ''key''=''value''`)
+	assert.Equal(s.T(), "", key)
+	assert.Equal(s.T(), "", value)
+
+	key, value, _ = parseSetStatement(`set '"key"'='"value"'`)
```

Comment:
Looks like we're only supporting single quotes for the outermost quote, why not support double quotes too?

Reply:
This is how the open source SQL client does it. Single quotes for the outermost quote and then you can use double quotes if you want quotes inside the key/value.